pinot-common/src/main/java/org/apache/pinot/common/utils/PinotDataType.java

```diff
@@ -92,6 +115,16 @@ public double toDouble(Object value) {
       return ((Byte) value).doubleValue();
     }
 
+    @Override
+    public boolean toBoolean(Object value) {
+      return (Byte) value > 0;
```

Comment:
Can we replace this by `return (Byte) value != 0`? I think the common convention (at least in c/c++ and a few other languages) is to treat zero as false and everything else as true. Same with other data types (character, integer, long, float) toBoolean function.

Reply:
Good suggestion, changed